packages/flutter/lib/src/material/stepper.dart

```diff
@@ -237,6 +238,9 @@ class Stepper extends StatefulWidget {
   /// The index into [steps] of the current step whose content is displayed.
   final int currentStep;
 
+  /// The padding around all Step Widgets
+  final EdgeInsets stepPadding;
```

Comment:
This property should be `EdgeInsets?` (nullable) and should document its default value.